src/main/java/controller/BlackJackGame.java

```diff
@@ -0,0 +1,170 @@
+        for (Player player : players.getPlayers()) {
+            String playerName = player.getPlayerName().getName();
+            String playerCards = convertUserToCards(player);
+            OutputView.printCardsHeldByPlayer(playerName, playerCards);
+        }
+    }
+
+    private static void askWantMoreCardToAllPlayers(Players players) {
+        players.getPlayers().forEach(BlackJackGame::askWantMoreCardToPlayer);
+    }
+
+    private static void askWantMoreCardToPlayer(Player player) {
+        if (player.isBust() || player.isBlackJack()) {
+            return;
+        }
+        chooseAnswer(player);
+    }
+
+    private static void chooseAnswer(Player player) {
+        String playerName = player.getPlayerName().getName();
+        OutputView.printDoYouWantOneMoreCard(playerName);
+        Answer answer = new Answer(InputView.inputAnswer());
+        if (answer.isYes()) {
+            distributeCardToPlayer(player);
+        }
+    }
+
+    private static void distributeCardToPlayer(Player player) {
+        Deck.distributeCard(player);
+        String playerName = player.getPlayerName().getName();
+        String playerCardsInfo = convertUserToCards(player);
+        OutputView.printCardsHeldByPlayer(playerName, playerCardsInfo);
+        askWantMoreCardToPlayer(player);
+    }
+
+    private static void distributeCardToDealer(Dealer dealer) {
+        if (dealer.hasSmallNumberLessThanRuleNumber()) {
+            OutputView.printDealerGetOneMoreCard();
+            Deck.distributeCard(dealer);
+            distributeCardToDealer(dealer);
+        }
+    }
+
+    private static void showDealerAndPlayersResult(Players players, Dealer dealer) {
+        showDealerResult(dealer);
+        showPlayersResult(players);
+    }
+
+    private static void showDealerResult(Dealer dealer) {
+        String dealerCardsInfo = convertUserToCards(dealer);
+        int dealerTotalCardNumber = dealer.calculateTotalCardNumber();
+        OutputView.printDealerResult(dealerCardsInfo, dealerTotalCardNumber);
+    }
+
+    private static void showPlayersResult(Players players) {
+        for (Player player : players.getPlayers()) {
+            String playerName = player.getPlayerName().getName();
+            String playerCardsInfo = convertUserToCards(player);
+            int totalCardNumber = player.calculateTotalCardNumber();
+            OutputView.printPlayerResult(playerName, playerCardsInfo, totalCardNumber);
+        }
+    }
+
+    private static String convertUserToCards(User user) {
+        return user.getCards().stream()
+                .map(card -> card.getType().getTypeName() + card.getSymbol().getSymbolName())
+                .collect(joining(SEPARATOR));
+    }
+
+    private static void showDealerAndPlayersProfit(Players players, Dealer dealer) {
+        PlayerProfitStatistics playerProfitStatistics = new PlayerProfitStatistics(players.producePlayersProfitStatistics(dealer));
```

Comment:
```suggestion
        PlayerProfitStatistics playerProfitStatistics = players.producePlayersProfitStatistics(dealer);
```
위와 같이 ``producePlayersProfitStatistics`` 메서드의 반환 타입을 ``PlayerProfitStatistics``로 하는 것이 낫지 않을까요?